### PR TITLE
Include social media links

### LINKED ISSUE #72: Include social media links
Hello guys,

I noticed that the link to documentation and github is duplicated in the header and buttons on the main page. 

I think we can improve the main page by including the social media links in the website header and keeping the documentation and github link in the main page buttons.

We can also change the name of the github button to Project.

Example:

image

### PR DESCRIPTION
**Pull request by priscila-qa**
Fixes #72

### Author/Contributor
- [ ] If documentation is needed for this change, has that been included in this pull request
- [ ] Remember to check if code coverage decrease, if so, please implement the necessary tests

### Reviewing Maintainer
- [ ] Label as `breaking` if this is a large fundamental change
- [ ] Label as either `automation`, `bug`, `documentation`, `enhancement`, `infrastructure`, or `performance`

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -23,14 +23,21 @@ module.exports = {
       },
       items: [
         {
-          to: 'docs/',
```
Comment (jhomarolo) [inline docusaurus.config.js:26]: Did you remove the docs links ? I think they should be there.

**Review by jhomarolo — CHANGES_REQUESTED** [review]
Could you please upload a printscreen here with the final result?

File: docusaurus.config.js (hunk 1 above)
Comment (priscila-qa) [reply docusaurus.config.js:26]: Yes, I removed it following the problem's suggestion (#72), because the "docs" are already in the "get started" button.

Comment (priscila-qa) [reply docusaurus.config.js:26]: > 

Sure.

**Comment by priscila-qa** [comment]
Results:

image
image

File: docusaurus.config.js (hunk 1 above)
Comment (priscila-qa) [reply docusaurus.config.js:26]: The replacement of the text by the social media icons, was a suggestion given by David in the fix comments.

Comment (jhomarolo) [reply docusaurus.config.js:26]: image

It seems the buttons are now well aligned, the space between the first and second buttons is not the same in the third and fourth buttons.

Comment (dalssoft) [reply docusaurus.config.js:26]: + 1

Comment (priscila-qa) [reply docusaurus.config.js:26]: The spacing has been corrected. 
What do you think? :) 

image

**Pull request merged**